packages/react/src/patterns/blocks/LeadSpaceBlock/README.md

```diff
@@ -0,0 +1,143 @@
+# Lead Space Block
+
+> The "Lead Space Block" pattern leverages the Content Block as the
+> container of this pattern, and allows to select media with link lists 
+> followed by a button.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### CSS
+
+```css
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/leadspace-block/index';
+```
+
+##### JS
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LeadSpaceBlock } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const title = 'Continuous delivery';
+  
+      const heading= text(
+        'Heading (required)',
+        'Innovate like a startup and scale for the enterprise '
+      );
+  
+      const copy = `Automate your software release process with continuous delivery (CD)—the most 
+      critical part of adopting DevOps. Build, test, and deploy code changes quickly, 
+      ensuring software is always ready for deployment.`;
+  
+      const mediaType = select(
+        'mediaType (optional)',
+        ['image', 'video', 'none'],
+        'image'
+      );
+  
+      const video = {
+        videoId: '0_uka1msg4',
+        showDescription: true,
+      };
+  
+      const image = {
+        heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+        image: {
+          sources: [
+            {
+              src: 'https://dummyimage.com/320x180/ee5396/161616&text=16:9',
+              breakpoint: 320,
+            },
+            {
+              src: 'https://dummyimage.com/400x225/ee5396/161616&text=16:9',
+              breakpoint: 400,
+            },
+            {
+              src: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+              breakpoint: 672,
+            },
+          ],
+          alt: 'Image alt text',
+          defaultSrc: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+        },
+      };
+  
+      const mediaData = mediaType === 'image' ? image : video;
+  
+      const linkListProps = {
+        heading: text('link list heading:', 'Featured products'),
+        items: object('link list items array', [
+          {
+            type: 'local',
+            copy: 'IBM Cloud Continuous Delivery',
+            cta: {
+              href: 'https://ibm.com',
+            },
+          },
+          {
+            type: 'local',
+            copy: 'UrbanCode',
+            cta: {
+              href: 'https://ibm.com',
+            },
+          },
+          {
+            type: 'local',
+            copy: 'View all products',
+            cta: {
+              href: 'https://ibm.com',
+            },
+          },
+        ]),
+      };
+
+  return (
+    <LeadSpaceBlock title={title} 
+                    heading={heading}
+                    copy={copy}
+                    mediaType={mediaType}
+                    mediaData={mediaData} 
+                    items={linkListProps}
```

Comment:
should add the button here as part of the example.

Reply:
added button props and changed the prop type as well.

Reply:
@greeshmareddyg this is still referencing button, should be `cta`.